### PR TITLE
Download json schemas from server

### PR DESCRIPTION
**Pull request by Fatme**
Should be merged after this PR https://github.com/Icenium/Ice/pull/2901

### CODE REVIEW

File: lib/commands/dev/prepackage.ts [hunk 1]
```diff
@@ -22,6 +23,8 @@ export class PrePackageCommand implements ICommand {
 			this.$templatesService.downloadProjectTemplates().wait();
 			this.$logger.info("Downloading item templates.");
 			this.$templatesService.downloadItemTemplates().wait();
+			this.$logger.info("Downloading json schemas");
```
Comment (teobugslayer) [inline lib/commands/dev/prepackage.ts]: This is user-visible message, right? In this case, I recommend we say "Downloading project schemas"

File: lib/json-schema/json-schema-loader.ts [hunk 2]
```diff
@@ -25,20 +28,41 @@ export class JsonSchemaLoader implements IJsonSchemaLoader {
...
+			var file = this.$fs.createWriteStream(filePath);
+			var fileEnd = this.$fs.futureFromEvent(file, "finish");
+
+			this.$httpClient.httpRequest({ url: zipUrl, pipeTo: file}).wait();
+			fileEnd.wait();
 
-			var schemas = _.values(this.schemas);
-			_.each(schemas, (schema: ISchema) => this.loadSchema(schema).wait());
+			this.$fs.unzip(filePath, this.schemasFolderPath).wait();
+			this.$fs.deleteFile(filePath).wait();
```
Comment (teobugslayer) [inline lib/json-schema/json-schema-loader.ts]: Can you use temp-generated file instead and save couple of code lines?

**Comment by teobugslayer** [comment]
Unit tests are failing :(

**Comment by tytodorov** [comment]
run ci

**Comment by Fatme** [comment]
I'll reopen it against release branch

**Comment by Fatme** [comment]
run ci

**Comment by Fatme** [comment]
run ci

**Comment by rosen-vladimirov** [comment]
14:21:11 /home/ubuntu/jenkins/workspace/ICE_CLI_PR/lib/json-schema/json-schema-loader.ts(38,19): error TS2346: Supplied parameters do not match any signature of call target.

**Comment by Fatme** [comment]
run ci

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**